cirq/circuits/qasm_output.py

```diff
@@ -69,6 +69,12 @@ def _unitary_(self) -> np.ndarray:
         ]
         return linalg.dot(*map(protocols.unitary, operations))
 
+    def __eq__(self, other):
+        return isinstance(other, QasmUGate) and \
```

Comment:
Use parens instead of `\` to combine lines.

Reply:
ack. I did that too many places, I'll review and replace.